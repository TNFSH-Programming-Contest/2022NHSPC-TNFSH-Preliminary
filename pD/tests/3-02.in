7 6
2 1 2 1 1 1 1
0 3 0 3 0 3
..T.T.
..T...
...TT.
.T....
......
..T..T
....T.
