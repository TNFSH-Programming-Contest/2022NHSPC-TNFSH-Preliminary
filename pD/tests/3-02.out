.AT.TA
..TA..
.A.TTA
.T.A..
.....A
.AT..T
...AT.
